composer.json

```diff
@@ -9,31 +9,37 @@
   "homepage": "https://theeventscalendar.com",
   "license": "GPL-2.0",
   "require-dev": {
-    "bordoni/phpass": "0.3.5",
     "automattic/vipwpcs": "^2.0",
+    "bordoni/phpass": "0.3.5",
+    "brianhenryie/strauss": "^0.11.0",
     "codeception/codeception": "^2.5.5",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
+    "lucatume/codeception-snapshot-assertions": "^0.2.4",
     "lucatume/function-mocker-le": "^1.0",
     "lucatume/wp-browser": "^3.0.14",
+    "phpunit/phpunit": "~6.0",
     "the-events-calendar/coding-standards": "dev-master",
     "the-events-calendar/tec-testing-facilities": "dev-master",
-    "phpunit/phpunit": "~6.0",
     "wp-cli/checksum-command": "1.0.5",
-    "wp-coding-standards/wpcs": "^2.1",
-    "lucatume/codeception-snapshot-assertions": "^0.2.4"
+    "wp-coding-standards/wpcs": "^2.1"
   },
   "minimum-stability": "dev",
   "prefer-stable": true,
   "config": {
     "preferred-install": "dist",
     "platform": {
-      "php": "7.3.33"
+      "php": "7.4.0"
```

Comment:
Minimum required version of PHP in the plugins is 7.3; the `platform.php` value should stay on `7.3.33`.